client/me/help/main.jsx

```diff
@@ -116,106 +115,109 @@ class Help extends React.PureComponent {
-								'Pick from our ever-growing list of free email courses to improve your knowledge.'
-							) }
-						</p>
-					</div>
-				</CompactCard>
-				<CompactCard
-					className="help__support-link"
-					href="https://learn.wordpress.com"
-					target="__blank"
-				>
-					<div className="help__support-link-section">
-						<h2 className="help__support-link-title">
-							{ this.props.translate( 'Self-guided online tutorial' ) }
-						</h2>
-						<p className="help__support-link-content">
-							{ this.props.translate(
-								'A step-by-step guide to getting familiar with the platform.'
-							) }
-						</p>
-					</div>
-				</CompactCard>
-				<CompactCard
-					className="help__support-link help__support-link-contact"
-					href="/help/contact/"
-				>
-					<div className="help__support-link-section">
-						<h2 className="help__support-link-title">{ this.props.translate( 'Get in touch' ) }</h2>
-						<p className="help__support-link-content">
+			<>
+				<h2 className="help__section-title">{ this.props.translate( 'More Resources' ) }</h2>
+				<div className="help__support-links">
+					{ this.getCoursesTeaser() }
+					<CompactCard
+						className="help__support-link"
+						href={ localizeUrl( 'https://wordpress.com/support/video-tutorials/' ) }
+						target="__blank"
+					>
+						<Gridicon icon="video" size={ 36 } />
+						<div className="help__support-link-section">
+							<h2 className="help__support-link-title">
+								{ this.props.translate( 'Video tutorials' ) }
+							</h2>
+							<p className="help__support-link-content">
+								{ this.props.translate(
+									'These short videos will demonstrate some of our most popular features.'
+								) }
+							</p>
+						</div>
+					</CompactCard>
+					<CompactCard
+						className="help__support-link"
+						href="https://dailypost.wordpress.com/blogging-university/"
+						target="__blank"
+					>
+						<Gridicon icon="mail" size={ 36 } />
+						<div className="help__support-link-section">
+							<h2 className="help__support-link-title">
+								{ this.props.translate( 'Email courses' ) }
+							</h2>
+							<p className="help__support-link-content">
+								{ this.props.translate(
+									'Pick from our ever-growing list of free email courses to improve your knowledge.'
+								) }
+							</p>
+						</div>
+					</CompactCard>
+					<CompactCard
+						className="help__support-link"
+						href="https://learn.wordpress.com"
+						target="__blank"
+					>
+						<Gridicon icon="list-ordered" size={ 36 } />
+						<div className="help__support-link-section">
+							<h2 className="help__support-link-title">{ this.props.translate( 'Guides' ) }</h2>
+							<p className="help__support-link-content">
+								{ this.props.translate(
+									'A step-by-step guide to getting familiar with the platform.'
+								) }
+							</p>
+						</div>
+					</CompactCard>
+				</div>
+			</>
+		);
+	};
+
+	getContactUs = () => {
```

Comment:
You can actually shorten this if you want by removing the brace and the return, and just leaving the parenthesis. Though I'm not sure if this adheres to our code style

Reply:
Nice! TIL :D